virtual_rainforest/core/constants.py

```diff
@@ -21,5 +21,11 @@ class CoreConsts(ConstantsDataclass):
     zero_Celsius: ClassVar[float] = constants.zero_Celsius
     """Conversion constant from Kelvin to Celsius (°)."""
 
-    placeholder: float = 123.4
-    """A placeholder configurable constant."""
+    depth_of_active_soil_layer: float = 0.25
+    """Depth of the biogeochemically active soil layer [m].
```

Comment:
Based on previous discussions, we set the top soil layer to 50cm. Would it be worth changing that to get a better representation of the active zone?

Reply:
I also wonder if that is actually a core constant? It is very soil microbe specific

Reply:
The reason this is in core constants is because both the `soil` and `litter` models use it, but agree that it is confusing when the abiotic model also defined a soil layer height

Reply:
The depth of the soil layers is defined in the core schemer and initialized by the plant model. We could add the 0.25m as an additional layer or replace it; for the `abiotic` model it does not matter what the values are.